Jamulus.pro

```diff
@@ -93,22 +93,31 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+        	message("Native i386 build")
+        	libjackname = libjack.lib
+        } else {
+        	message("Native x86_64 build")
+		    libjackname = libjack64.lib
+		    !exists("C:/Program Files (86)/JACK2/include/jack/jack.h") {
+            	message("Warning: jack.h was not found at the usual place, install JACK2 32 bits to cross compile.")
+         	}	
         }
+          !exists("C:/Program Files/JACK2/include/jack/jack.h") {
```

Comment:
This check appears to be happening regardless of whether on 32 or 64bit.

Reply:
That's right. `jack.h` is at the same place whether you are in 32 bits or 64 bits Windows OS.
64 bits is  - > `C:/Program Files/JACK2/include/jack/jack.h` + eventually if 32 bits is installed `C:/Program Files (86)/JACK2/include/jack/jack.h`
where 32 bits is "only" -> `C:/Program Files/JACK2/include/jack/jack.h`